boards/st/stm32h750b_dk/stm32h750b_dk.yaml

```diff
@@ -9,7 +9,6 @@ toolchain:
 ram: 1024
 flash: 128
 supported:
-  - arduino_gpio
```

Comment:
I see that STM32H750B_DK has an `arduino_connector_r3.dtsi` file though; is this a mistake?
https://github.com/zephyrproject-rtos/zephyr/blob/874e4e2e199c138d73f3acfca9d9f7d90f79666b/boards/st/stm32h750b_dk/arduino_r3_connector.dtsi#L7-L36

Reply:
Indeed. The issue was that it wasn't included. Fixed